ipv-api/src/main/java/uk/gov/di/authentication/ipv/helpers/IPVCallbackHelper.java

```diff
@@ -222,6 +222,8 @@ public AuthenticationSuccessResponse generateReturnCodeAuthenticationResponse(
 
         cloudwatchMetricsService.incrementSignInByClient(
                 orchSession.getIsNewAccount(), clientId, clientName, isTestJourney);
+        cloudwatchMetricsService.incrementCounter(
+                "orchIdentityJourneyCompleted", Map.of("clientName", clientName));
```

Comment:
are you sure we won't need any other dimensions? e.g. maybe useful to have clientId there in case we want to filter for it within the dashboard because the client names can be quite long?